planning/behavior_path_planner/autoware_behavior_path_static_obstacle_avoidance_module/src/utils.cpp

```diff
@@ -749,6 +794,17 @@ bool isObviousAvoidanceTarget(
   return false;
 }
 
+/**
+ * @brief this function includes some conditions which apply to both vehicle and non-vehicle object.
+ * @param object data.
+ * @param current reference path.
+ * @param object detection range.
+ * @param distance between object and goal point.
```

Comment:
Explanation for `const Point & ego_pos` is missing.

```suggestion
 * @param distance between object and goal point.
 * @param ego position.
```